### PR TITLE
Practical Part 2

### PR DESCRIPTION
**Pull request by MaHmOuDeB**
Here are all the files that are modified/added for the second part of the practical part

### CODE REVIEW

File: demos/CMakeLists.txt [hunk 1]
```diff
@@ -35,3 +35,5 @@ create_demo(Demo_CSG "Demo CSG")
 create_demo(Demo_DoF "Demo DoF")
 create_demo(Demo_VR "Demo Virtual Reality")
 create_demo(Demo_Texturing "Demo Texturing")
+
```
Comment (ereator) [inline demos/CMakeLists.txt]: This file should not be changed, and as consequence should not be present in the pull request.
The easiest way to undo all the changes - is to _revert_ this file, for example, in GitHub Desktop application.

File: modules/core/TextureRings.cpp [hunk 2]
```diff
@@ -1,26 +1,42 @@
-//
-//  Rings.cpp
-//  Demo_AreaLight
-//
 //  Created by Mahmoud El Bergui on 18.04.22.
 //
 
 #include "TextureRings.h"
+#include"PerlinNoise.h"
```
Comment (ereator) [inline modules/core/TextureRings.cpp]: Do you use Perlin Noise in this texture?

Comment (ereator) [inline modules/core/TextureRings.cpp]: Do you use Perlin Noise in this texture?

**Review by ereator — CHANGES_REQUESTED** [review]

File: demos/CMakeLists.txt (hunk 1 above)
Comment (MaHmOuDeB) [reply demos/CMakeLists.txt]: I reverted the file to the last time when you modified

File: modules/core/TextureRings.cpp (hunk 2 above)
Comment (MaHmOuDeB) [reply modules/core/TextureRings.cpp]: No I don't I was just doing some tests and forgot to put take it off

**Pull request closed without merging**